tests/test_harvester_gsi_satellite_radiance.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
+
+"""Unit tests for gsi_satellite_radiance.py
+"""
+
+import os
+from pathlib import Path
+
+from score_hv import hv_registry
+from score_hv.harvester_base import harvest
+
+PYTEST_CALLING_DIR = Path(__file__).parent.resolve()
+FIT_FILE_PATH = os.path.join(PYTEST_CALLING_DIR, 'data',
+                             'gsistats.1979032100_control')
+                             
+VALID_CONFIG_DICT = {'harvester_name': hv_registry.GSI_RADIANCE_CHANNEL,
+                     'filename': FIT_FILE_PATH,
+                     'variables': ('var',
+                                   'varch_cld',
+                                   'use',
+                                   'ermax',
+                                   'b_rad',
+                                   'pg_rad',
+                                   'icld_det',
+                                   'icloud',
+                                   'iaeros',
+                                   'bias_correction_coefficients'),
+                     'statistics': ('nobs_used',
+                                    'nobs_tossed',
+                                    'variance',
+                                    'bias_pre_corr',
+                                    'bias_post_corr',
+                                    'penalty',
+                                    'sqrt_bias',
+                                    'std')
+                    }
+    
+def test_bad_config():
```

Comment:
could you please add a comment here about what makes this config "bad" so it's clear what it's actually testing?

nice to test for the exception!

Reply:
clarifying, that it appears that it uses "variances" instead of "variance", but my first guess on what this was testing was the key name for the harvester name itself (so a failure in the harvest function not in the gsi radiances file). that's why it would be nice to make this clearer what it's checking for.

Reply:
added docstring "test that a misconfigured config_dict does not result in data being returned by the harvester"